From 9739af046a243f79ef62f5a2b1c5aad85b401ac0 Mon Sep 17 00:00:00 2001
From: "Nguyen Van Tu [FGA.AIS]" <TuNV16@fsoft.com.vn>
Date: Thu, 8 Nov 2018 09:20:06 +0700
Subject: [PATCH 192/628] ARM: dts: iwg22m: remove msiof2_sync pin

msiof2_sync is shared with CAN.
remove it to prevent conflict with CAN.
With current testing (write/read), msiof2 still work OK without msiof2_sync
Futher testing/investigating may be needed

Signed-off-by: khanhluu <khanh.luu.xw@rvc.renesas.com>
Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNV25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Nguyen Van Tu [FGA.AIS] <TuNV16@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7745-iwg22m.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7745-iwg22m.dtsi b/arch/arm/boot/dts/r8a7745-iwg22m.dtsi
index 190137a..a89e784 100644
--- a/arch/arm/boot/dts/r8a7745-iwg22m.dtsi
+++ b/arch/arm/boot/dts/r8a7745-iwg22m.dtsi
@@ -112,7 +112,7 @@
 
 	msiof2_pins: spi3 {
 		renesas,groups = "msiof2_clk", "msiof2_tx", "msiof2_rx",
-				"msiof2_ss2", "msiof2_sync";
+				"msiof2_ss2";
 		renesas,function = "msiof2";
 	};
 
-- 
2.7.4

